crates/smartmodule-development-kit/src/main.rs

```diff
@@ -23,6 +24,6 @@ fn main() -> Result<()> {
 }
 
 pub(crate) fn read_bytes_from_path(path: &PathBuf) -> Result<Vec<u8>> {
-    println!("loading module at: {}", path.display());
+    debug!("loading module at: {}", path.display());
```

Comment:
use standard tracing format.  so it should be `debug(path = path.display(),"loading module")`

Reply:
updated it. Thanks